File: Unidad4/C#/Programa1.cs
using System;


public class ProgramaBases
{

    public static void Main(String[] args)
    {
        int[] edades = new int [] { 17, 19, 18, 18, 18 };


        for (int i = 0; i < edades.Length; i++)
        {
            Console.WriteLine("indice:" +i+" es:" + edades[i]);
            if (edades[i] >=18)

            Console.WriteLine("Eres mayor");

            else

            Console.WriteLine("Eres menor");


        }

    }
}

File: Unidad4/Java/Programa1.js

package com.mycompany.programa1;

public class Programa1 {

    public static void main(String[] args) {
        int[] edades = { 17, 19, 18, 18, 18 };
        
        for (int i = 0; i < edades.length; i++) {
            System.out.println("indice: " + i + " es: " + edades[i]);
            
            if (edades[i] >= 18)
                System.out.println("Eres Mayor");
            else
                System.out.println("Eres Menor");
        }
    }
}

